index.html

```diff
@@ -3,10 +3,10 @@
 
 <head>
   <meta charset="utf-8">
-  <title>Some Student</title>
+  <title>Dustin Walker</title>
 
   <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:200,300" rel="stylesheet">
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.css">
```

Comment:
Here you should have all three stylesheets. You made the correct link to `./styles/style.css` but now the icons are no longer working 😢